prepare for 0.13.0 release

skatkov opened the pull request:
Want to cut a 0.13.0 release for Rubygems.org

also one line that I forgot to add to fix testsuit.

jujustayfly commented:
Sorry but why do we want to release a new version of this? Has anything changed?

skatkov commented:
@jujustayfly good question.

We've changed minimal ruby dependency, changelog contained a lot of changes that have not been released and we've introduced activesupport as a dependency.

https://github.com/cheddar-me/pbbuilder/pull/18/files

I also would like to write down all the steps required to release this gem in CONTRIBUTION.md

But we can, ofcouse, wait until caching feature will be finished also.
